deploy-manage/security/using-kibana-with-security.md

```diff
@@ -32,77 +30,9 @@ Set an encryption key so that sessions are not invalidated. You can optionally c
-    ```
-
-    ::::{tip}
-    For more information on Basic Authentication and additional methods of authenticating {{kib}} users, see [Authentication](/deploy-manage/users-roles/cluster-or-deployment-auth/user-authentication.md).
-    ::::
-
-3. Grant users access to the indices that they will be working with in {{kib}}.
-
-    ::::{tip}
-    You can define as many different roles for your {{kib}} users as you need.
-    ::::
+3. Restart {{kib}}.
 
-
-    For example, create roles that have `read` and `view_index_metadata` privileges on specific data views. For more information, see [User authorization](/deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md).
-
-4. Log out of {{kib}} and verify that you can log in as a normal user. If you are running {{kib}} locally, go to `https://localhost:5601` and enter the credentials for a user you’ve assigned a {{kib}} user role. For example, you could log in as the user `jacknich`.
-
-    ::::{note}
-    This must be a user who has been assigned [Kibana privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md). {{kib}} server credentials (the built-in `kibana_system` user) should only be used internally by the {{kib}} server.
-    ::::
-
-## Additional Kibana security configurations [Security-production-considerations]
-
-To secure your {{kib}} installation in production, consider these high-priority topics to ensure that only authorized users can access {{kib}}.
-
-### Enable SSL/TLS [enabling-ssl]
-
-You should use SSL/TLS encryption to ensure that traffic between browsers and the {{kib}} server cannot be viewed or tampered with by third parties. See [encrypt HTTP client communications for {{kib}}](/deploy-manage/security/set-up-basic-security-plus-https.md#encrypt-kibana-http).
-
-### Enabling mutual TLS between {{kib}} and {{es}}
-
-Refer to [](/deploy-manage/security/secure-http-communications.md#elasticsearch-mutual-tls) for information on how to enable mutual TLS between {{kib}} and {{es}}.
-
-### Use {{stack}} {{security-features}} [configuring-kibana-shield]
-
-You can use {{stack}} {{security-features}} to control what {{es}} data users can access through {{kib}}.
-
-When {{security-features}} are enabled, {{kib}} users have to log in. They must have a role granting [{{kib}} privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md) and access to the indices that they will be working with in {{kib}}.
-
-If a user loads a {{kib}} dashboard that accesses data in an index that they are not authorized to view, they get an error that indicates the index does not exist.
-
-For more information on granting access to {{kib}}, see [](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md).
-
-
-### Use secure HTTP headers [configuring-security-headers]
+## Use secure HTTP headers [configuring-security-headers]
 
 The {{kib}} server can instruct browsers to enable additional security controls using HTTP headers.
```

Comment:
line 39 - I'd just spell out "strict transport security" OR use code formatting and call it a header